src/qt/coincontroldialog.cpp

```diff
@@ -433,8 +433,7 @@ void CoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
         {
             CTxOut txout(amount, (CScript)std::vector<unsigned char>(24, 0));
             txDummy.vout.push_back(txout);
-            if (txout.IsDust(dustRelayFee))
-               fDust = true;
+            fDust = IsDust(txout, ::dustRelayFee);
```

Comment:
should change to `fDust =| IsDust(txout, ::dustRelayFee);` to maintain the logic.